pwdtk/auth_backends.py

```diff
@@ -44,20 +64,17 @@ def authenticate(self, request, username=None, password=None, **kwargs):
                     pwdtk_data.locked = False
                     pwdtk_data.must_renew = must_renew
                     pwdtk_data.save()
+                if must_renew and not kwargs.get("ignore_must_renew"):
+                    raise PwdtkForceRenewException(pwdtk_data)
                 return user
             else:
```

Comment:
why the else statement.

recommendation if (the if statement contains an unconditianal return, to get rid of the else and unindent the next commands.


```
if condition:
   bla
   return something

do_something_else
```

Reply:
will do